extensions/blocks/slideshow/slideshow.php

```diff
@@ -64,7 +64,7 @@ function render_amp( $attr ) {
 	);
 	$classes  = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attr, $extras );
 
-	return sprintf(
+	return empty( $ids ) ? null : sprintf(
```

Comment:
That empty check should probably happen further up the function to bail early. According to function docs it also expects to always return a string.

Reply:
Sure, I return now `''`. I think it is preferred than having a mixed return?